src/dialog/Dialog.ts

```diff
@@ -65,6 +67,22 @@ export default class Dialog extends DialogBase<DialogProperties> {
 		!this.properties.modal && this._onCloseClick();
 	}
 
+	private _onKeyUp(event: KeyboardEvent) {
+		if (event.keyCode === Keys.Escape) {
```

Comment:
we generally use `event.which` in widgets. There's no difference for the escape key, but it's nice to standardize things.

Reply:
I copied this from `titlepane`, happy to change as we should encourage consistency.

Reply:
raised: https://github.com/dojo/widgets/issues/341 to address

Reply:
haha didn't even notice `titlepane` still had keyCode 😆

Reply:
I was also debating making the switch over to `event.key`, since I think the support is pretty much there now.

Reply:
feel free to add a comment on https://github.com/dojo/widgets/issues/341